### ISSUE TITLE
Installation is failed on Linux x64 with version 0.47.1

### ISSUE THREAD

**Issue by fabasoad** [lindell/multi-gitter#373]
**Describe the bug**
I think the problem is that there is no version in file name for tar.gz assets in the latest release. So, installation script failed.

To compare, here are the assets of 0.47.1 release:

![Screenshot 2023-07-16 at 14 16 53](https://github.com/lindell/multi-gitter/assets/2127723/74153a62-45e3-4ef9-87ff-2dbfbd7bbc9c)

and here are the assets of 0.47.0 release:

![Screenshot 2023-07-16 at 14 17 39](https://github.com/lindell/multi-gitter/assets/2127723/acbc4570-df73-458f-8145-cd98a59d3603)

Notice version in a file name.

**To Reproduce**
1. Run the script below.

```shell
curl -s https://raw.githubusercontent.com/lindell/multi-gitter/master/install.sh | sh -s -- -d v0.47.1
```

**Expected behavior**
`multi-gitter` should be installed without any errors.

**Additional context**

```shell
lindell/multi-gitter info checking GitHub for tag 'v0.47.1'
lindell/multi-gitter debug http_download https://github.com/lindell/multi-gitter/releases/v0.47.1
lindell/multi-gitter info found version: 0.47.1 for v0.47.1/Linux/x86_64
lindell/multi-gitter debug downloading files into /tmp/tmp.oG2Wk42K9D
lindell/multi-gitter debug http_download https://github.com/lindell/multi-gitter/releases/download/v0.47.1/multi-gitter_0.47.1_Linux_x86_64.tar.gz
lindell/multi-gitter debug http_download_curl received HTTP status 404
```

**Comment by lindell** [comment]
Thanks for the report. It was caused by updating a deprecated field in goreleaser. I should be fixed in 0.47.3. Please confirm.

**Comment by fabasoad** [comment]
`0.47.3` works 👍 Thank you!
